pkg/config/spec.go

```diff
@@ -221,6 +222,10 @@ func NewUpgradeSpec(cfg *Config) (*v1.UpgradeSpec, error) {
 	var recLabel, recFs, recMnt string
 	var active, passive, recovery v1.Image
 
+	active.Source = v1.NewEmptySrc()
```

Comment:
this only affect normal upgrades, which dont have any issues with nil access :D

Fix should og unto `NewUpgradeUkiSpec`

Reply:
right indeed.. I've just followed the panic here, did realize afterwards that we were actually skipping the UKI upgrade command and going into normal mode